.gitignore

```diff
@@ -302,3 +302,6 @@ output.png
 .reference-assets.lock
 output.svg
 formatted.gbr
+
+# Local test files
+test/local/
```

Comment:
Why are you including this entry?

Reply:
This was a folder I was using for my own test scripts, but the .gitignore can be reverted if it's just clutter